src/student_success_tool/analysis/pdp/dataio.py

```diff
@@ -1,477 +0,0 @@
-import functools as ft
```

Comment:
functionality here has been broken out into separate modules in the new top-level `dataio` subpackage